shared/components/BaseButton/package.json

```diff
@@ -4,6 +4,9 @@
   "private": true,
   "main": "",
   "module": "",
+  "dependencies": {
+    "@tds/core-a11y-content": "^1.0.0"
```

Comment:
Since BaseButton is a devDependency of Button and ButtonLink, BaseButton will be built, and not downloaded as a separate module. Perhaps core-a11y-content should be a dependency of ButtonLink and Button that gets passed to BaseButton, that way consumers of Button and ButtonLink can use core-a11y-content as a shared module, rather than something pre-compiled.

Alternatively, you could deploy BaseButton as an npm module.